docs/docs/adding-tags-and-categories-to-blog-posts.md

```diff
@@ -193,41 +193,40 @@ exports.createPages = ({ actions, graphql }) => {
         }
       }
     }
-  `).then(result => {
-    if (result.errors) {
-      return Promise.reject(result.errors)
-    }
+  `)
 
-    const posts = result.data.allMarkdownRemark.edges
+  // handle errors
+  if (result.errors) throw new Error(result.errors)
 
-    // Create post detail pages
-    posts.forEach(({ node }) => {
-      createPage({
-        path: node.fields.slug,
-        component: blogPostTemplate,
-      })
-    })
+  const posts = result.data.allMarkdownRemark.edges
 
-    // Tag pages:
-    let tags = []
-    // Iterate through each post, putting all found tags into `tags`
-    _.each(posts, edge => {
-      if (_.get(edge, "node.frontmatter.tags")) {
-        tags = tags.concat(edge.node.frontmatter.tags)
-      }
+  // Create post detail pages
+  posts.forEach(({ node }) => {
+    createPage({
+      path: node.fields.slug,
+      component: blogPostTemplate,
     })
-    // Eliminate duplicate tags
-    tags = _.uniq(tags)
-
-    // Make tag pages
-    tags.forEach(tag => {
-      createPage({
-        path: `/tags/${_.kebabCase(tag)}/`,
-        component: tagTemplate,
-        context: {
-          tag,
-        },
-      })
+  })
+
+  // Tag pages:
+  let tags = []
+  // Iterate through each post, putting all found tags into `tags`
+  _.each(posts, edge => {
```

Comment:
Lets convert this instance of lodash. The other instances like `_.uniq` bring value as `tags = Array.from(new Set(tags))` would probably be hard to read.

```suggestion
  posts.forEach(edge => {
```

Reply:
As a side note, I submitted another pull request to simplify this page #15576